create-publication-from-scopus/src/test/resources/expectedResults/expectedMathMlContent.txt

```diff
@@ -0,0 +1 @@
+Non-factorizable contributions to $\overline{B} _{d} ^{\mathrm{0}}$ → $D _{s}$ (*) $\overline{D} _{s}$ (*)
```

Comment:
Can you provide the original article this is extracted from? I need an actual comparison as I believe we want something like https://aip.scitation.org/doi/10.1063/1.2163761